pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1519,10 +1519,11 @@ type ObjectEndpoints struct {
 	Secure []string `json:"secure"`
 }
 
-// CephObjectStoreUser represents a Ceph Object Store Gateway User
 // +genclient
 // +genclient:noStatus
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// CephObjectStoreUser represents a Ceph Object Store Gateway User
 // +kubebuilder:resource:shortName=rcou;objectuser
```

Comment:
Are we sure that the empty lines between the comments, isn't breaking any of the client generation? Is the Golang AST/ code gen smart enough about it?

Reply:
After tests seems everything is working as we expected